src/wallet/bdb.cpp

```diff
@@ -342,7 +330,8 @@ BerkeleyBatch::BerkeleyBatch(BerkeleyDatabase& database, const char* pszMode, bo
 
     {
         LOCK(cs_db);
-        if (!env->Open(false /* retry */))
+        bilingual_str open_err;
+        if (!env->Open(open_err))
```

Comment:
style-nit: Could upgrade to the new style while touching the line
```suggestion
        if (!env->Open(open_err)) {
```